syntax = "proto3";

package df.conf.proto;

import "graph.proto";

message DfConfig {
    repeated Graph graphs = 1;
}